public/js/whiteboard.js

```diff
@@ -1,4 +1,9 @@
-/* global moment, io */
+/* global moment, io, SunCalc */
+
+const longitude = -73.675770;
+const latitude = 42.729270;
```

Comment:
Is there a simple way we can put these in a server-side env var?

Reply:
Only way with no other changes would be to make a JSON endpoint to serve these variables, but that's a bit janky. Probably better to just incorporate webpack or something into the process and have the variables get filled in at build time.

Reply:
I think that's a good idea for the future. For now, this works.